src/main.rs

```diff
@@ -164,6 +177,12 @@ pub fn parse_args() -> MiniserveConfig {
         None
     };
 
+    let mut random_route = None;
```

Comment:
Can you make this a bit like https://github.com/svenstaro/miniserve/pull/26/files#diff-639fbc4ef05b315af92b4d836c31b023R158?

This way, you don't have the mutable variable around.